docker/Dockerfile.base.arm64

```diff
@@ -3,7 +3,7 @@
 # libraries.
 #
 
-FROM arm64v8/ubuntu:18.04
+FROM arm64v8/ubuntu:16.04
```

Comment:
how about using some fixed one like the way its done for amd? Below is for amd:
```
FROM openebs/cstor-ubuntu:xenial-20190515
```

Reply:
cc: @kmova

Reply:
@mynktl  -- can you try with this please. `openebs/arm64v8-ubuntu:xenial-20200326`

Reply:
Thank you @kmova for creating ARM image.
trivy scan result with above image is as below:
```bash
openebs/cstor-pool-arm64:ci (ubuntu 16.04)
==========================================
Total: 270 (UNKNOWN: 0, LOW: 227, MEDIUM: 43, HIGH: 0, CRITICAL: 0)
```